src/modules/navigator/mission.h

```diff
@@ -91,6 +91,9 @@ class Mission : public MissionBlock, public ModuleParams
 	bool get_mission_finished() const { return _mission_type == MISSION_TYPE_NONE; }
 	bool get_mission_changed() const { return _mission_changed ; }
 	bool get_mission_waypoints_changed() const { return _mission_waypoints_changed ; }
+	float get_landing_lat() { return _landing_lat; }
```

Comment:
```suggestion
	double get_landing_lat() { return _landing_lat; }
```
And in `src/modules/navigator/navigator.h` as well.